resize-observer/eventloop.html

```diff
@@ -0,0 +1,259 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="./resources/resizeTestHelper.js"></script>
+<style>
+  div {
+    border: 1px dotted gray
+  }
+</style>
+<p>ResizeObserver notification event loop tests</p>
+<div id="target1" style="width:100px;height:100px;">t1
+</div>
+<div id="container">
+  <div id="a1" style="width:100px;height:100px">
+    <div id="a2" style="width:100px;height:100px">
+    </div>
+  </div>
+  <div id="b1" style="width:100px;height:100px">
+    <div id="b2" style="width:100px;height:100px">
+    </div>
+  </div>
+</div>
+<script>
+'use strict';
+
+let t1 = document.querySelector('#target1');
+
+// allow uncaught exception because ResizeObserver posts exceptions
+// to window error handler when limit is exceeded.
+// This codepath is tested in this file.
+
+setup({allow_uncaught_exception: true});
+
+function template() {
+  let helper = new ResizeTestHelper(
+    "test0: title",
+  [
+    {
+      setup: observer => {
+      },
+      notify: (entries, observer) => {
+        return true;  // Delay next step
+      }
+    }
+  ]);
+  return helper.start();
+}
+
+var onErrorCalled = false;
+
+window.onerror = err => {
```

Comment:
`err` will be bound to an event, and in any case it's not used, so `_` or `()` would be more clear.

Reply:
will do.